Resources/Private/Templates/Backend/BuildForm.html

```diff
@@ -33,6 +33,7 @@
 			<f:render section="Checkbox" arguments="{name: 'backend'}" />
 			<f:render section="Checkbox" arguments="{name: 'controllers'}" />
 			<f:render section="Checkbox" arguments="{name: 'vhs', checked: 'checked'}" />
+			<f:render section="Checkbox" arguments="{name: 'fluidcontentCore', checked: 'fluidcontentCore'}" />
```

Comment:
I think either remove the `checked` parameter here or use a variable coming from the View which you then assign from the outside. The value should be either `NULL` or `'checked'` and should be "checked" by default only when `fluidcontent_core` is installed.